core/Cargo.toml

```
@@ -40,7 +40,7 @@ chrono = { workspace = true, features = ["clock"] }
 derivative = "2.2.0"
 futures = "0.3.28"
 hex = "0.4"
-hpke-dispatch = "0.5.1"
+hpke-dispatch = { version = "0.5.0", features = ["serde"] }
```

We may as well leave the version requirement as-is. (note that the lockfile is still pointing at 0.5.1)
```suggestion
hpke-dispatch = { version = "0.5.1", features = ["serde"] }
```